packages/utils/time.test.ts

```
@@ -1,13 +1,114 @@
-describe('time', () => {
-	test.each([
-		[0, '0:00'],
-		[2500, '0:02'],
-		[Minute * 3, '3:00'],
-		[Hour + Minute * 3, '63:00'],
-		[Hour + Minute * 3 + Second, '63:01'],
-	])('should support formatting durations', (input, expected) => {
-		expect(formatMsToDurationCompat(input)).toBe(expected);
-	});
+import {
+  formatMsToDurationCompat,
```

ESLint

[error] Expected indentation of 1 tab but found 2 spaces.

(`@typescript-eslint/indent`)

---

[error] Expected indentation of 1 tab but found 2 spaces.

(indent)